_raml/apps/orders/schema/get_order_orderid_documents.schema.json

```diff
@@ -0,0 +1,23 @@
+{
+  "$schema": "http://json-schema.org/draft-03/schema",
+  "title": "Get Orders OrderId Documents Schema",
+  "type": "object",
+  "properties": {
+    "results": {
+      "type": "integer",
+      "description": "The number of orders in total."
```

Comment:
This is not the number of orders but the number of documents (finalized credit notes and invoices) for a single order.